municipalities/beek-design-tokens/src/component/conduction/card-wrapper.tokens.json

```diff
@@ -0,0 +1,33 @@
+{

```

Comment:
I expect the cards to look a bit more like the cards on https://www.gemeentebeek.nl/bestuur